libbeat/docs/tab-widgets/ingest-data-widget.asciidoc

```diff
@@ -0,0 +1,40 @@
+++++
```

Comment:
There's a bug in the widget code (or maybe it's a problem with having nested tabs...not sure). To reproduce, click Application logs then click Data collection modules.

![image](https://user-images.githubusercontent.com/14206422/104253617-cf1b5780-5429-11eb-9572-3f43d7a1a99b.png)